docs/tutorials/debugging.md

```diff
@@ -119,20 +148,32 @@ def f(x, y):
   z = x / y
   breakpoint_if_nonfinite(z)
   return z
-f(2., 0.) # ==> Pauses during execution
+
+f(2., 1.) # ==> No breakpoint
 ```
 
-![JAX debugger](../_static/debugger.gif)
+```{code-cell}
+:tags: [skip-execution]
+
+f(2., 0.) # ==> Pauses during execution
+```
 
 ## JAX `debug.callback` for more control during debugging
 
-As mentioned in the beginning, {func}`jax.debug.print` is a small wrapper around {func}`jax.debug.callback`. The {func}`jax.debug.callback` method allows you to have greater control over string formatting and the debugging output, like printing or plotting. It is compatible with {func}`jax.jit`, {func}`jax.vmap`, {func}`jax.grad` and other transformations (refer to the {ref}`external-callbacks-flavors-of-callback` table in {ref]`external-callbacks` for more information).
+Both {func}`jax.debug.print` and {func}`jax.debug.breakpoint` are implemented in terms of
+the more flexible {func}`jax.debug.callback`, which gives greater control over the
+host-side logic executed during the callback.
```

Comment:
```suggestion
host-side logic executed via a Python callback.
```
"the callback" is confusing because you haven't introduced any callback yet I think?

Reply:
Done